app/src/main/res/drawable/ic_is_dashed_square.xml

```diff
@@ -1,8 +1,8 @@
-<vector android:height="24dp"
-    android:viewportHeight="61.382"
-    android:viewportWidth="61.382"
-    android:width="24dp"
-    xmlns:android="http://schemas.android.com/apk/res/android">
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
WARNING: [UnusedResources] The resource `R.drawable.ic_is_dashed_square` appears to be unused
 Unused resources make applications larger and slow down builds.